### PR TITLE
Clean up Gemfile; only use NewRelic in production

### PR DESCRIPTION
**Pull request by awead**
Puts New Relic in production only, so we don't run into problems locally, then configures Travis to run our CI build and check for Rubocop violations; lastly, fix the outstanding violations! 🚓

### CODE REVIEW

**Comment by awead** [comment]
@cam156 I think this PRs is stepping a bit on your PR's toes, but I need to do this to finish setting up Travis. It was failing when I submitted other PRs. I think if you rebase yours against this, it should work. The Gemfile will be very different because I re-arranged everything for clarity. Hope that's ok.

**Comment by cam156** [comment]
@awead I have travis about half set up on my box, so we were both working on the same thing.  WAS there a reason not to merge my PR? I do not see any comments

File: config/deploy.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # frozen_string_literal: true
+
 # lock '3.6'
 
 # Set assets roles to occur on jobs as well as web
-set :assets_role, [:web, :job]
+set :assets_role, %i[web job]
```
Comment (cam156) [inline config/deploy.rb]: We need to figure out how to get rubocop running the correct style guide version

Comment (cam156) [inline config/deploy.rb]: if this PR gets merged reverting the rubocop comit and rerunning rubocop should work: https://github.com/psu-stewardship/guides/pull/3/files

**Comment by cam156** [comment]
@awead I am going to dismiss my review so it does not block you while I am away.

**Pull request merged**